src/hotspot/share/utilities/vmError.cpp

```diff
@@ -32,7 +32,13 @@
 #include "compiler/disassembler.hpp"
 #include "gc/shared/gcConfig.hpp"
 #include "gc/shared/gcLogPrecious.hpp"
+#if INCLUDE_JFR
+#include "jfr/jfr.hpp"
+#endif
 #include "jvm.h"
+#if INCLUDE_JVMCI
+#include "jvmci/jvmci.hpp"
+#endif
```

Comment:
These changes shouldn't be made.  Conditional includes go at the end.

Reply:
Noted, reverted in a27bef538f1b5cb3465e247798319a2d4a463102